dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/filter/AccessLogFilter.java

```diff
@@ -115,35 +115,41 @@ public Result invoke(Invoker<?> invoker, Invocation inv) throws RpcException {
     private void log(String accessLog, AccessLogData accessLogData) {
         Set<AccessLogData> logSet = LOG_ENTRIES.computeIfAbsent(accessLog, k -> new ConcurrentHashSet<>());
 
-        if (logSet.size() < LOG_MAX_BUFFER) {
+        if (logSet.size() >= LOG_MAX_BUFFER) {
```

Comment:
Shouldn't this comparator be "<", which means we can only add new item to logSet when it's size does not exceed `LOG_MAX_BUFFER `?

Reply:
You are right! I will fix this.